tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md

```diff
@@ -125,11 +125,11 @@ In this step, you set project attributes and choose to add further optional conf
 
 In this step, you define where you want the UI project to be deployed to as a runnable application.
 
-<!-- border -->![UI Wizard: Configure Deployment Settings](bas-appgen-deploy-config.png)
+![UI Wizard: Configure Deployment Settings](bas-appgen-deploy-config.png)
 
 1. Leave the default `ABAP` as **target** platform and as **Destination** the one you have created earlier (see [Connect SAP Business Application Studio and SAP S/4HANA Cloud tenant](abap-custom-ui-bas-connect-s4hc)).
 
-2. Enter a **Name** that represents the UI5 ABAP repository the application will be deployed to. Example: `YY1_BONUSPLAN`. This repository name will be visible as your **Custom UI App ID** in your SAP S/4HANA Cloud tenant.
+2. Enter the **SAPUI5 ABAP Repository** Name that will be created and the application be deployed to. Example: `YY1_BONUSPLAN`. This repository name will be visible as your **Custom UI App ID** in your SAP S/4HANA Cloud tenant.
```

Comment:
Enter a name for the **SAPUI5 ABAP Repository**. This is the repository that will be created and where the application will be deployed.